files/es/learn/common_questions/what_is_a_url/index.md

```diff
@@ -106,39 +106,44 @@ Veamos algunos ejemplos para aclarar esto.
 #### Ejemplos de URL absolutas
 
 - URL Completa (la misma que usamos antes)
-  - : &nbsp;
-      ```
-      https://developer.mozilla.org/en-US/docs/Learn
-      ```
+
+  ```
+  https://developer.mozilla.org/en-US/docs/Learn
+  ```
+
```

Comment:
This breaks the definition list format and converts into a list format, not sure if there's another way to keeo the format

```suggestion
  - : &nbsp;
      ```
      https://developer.mozilla.org/en-US/docs/Learn
      ```
```

Reply:
Upstream, this was a table rather than a definition list

Reply:
I went ahead and converted it back to a table like upstream